packages/compass-e2e-tests/tests/collection-documents-tab.test.ts

```diff
@@ -297,7 +297,7 @@ describe('Collection documents tab', function () {
       useBuilders: true,
     });
 
-    expect(text).to.equal(`import static com.mongodb.client.model.Filters.eq;
+    expect(text).to.equal(`import org.bson.Document;
```

Comment:
This was actually a bug before. The UI code and runTranspiler had  two different copies of the logic around whether the useBuilders checkbox should be there and taken into account. So we displayed the use builders syntax checkbox and the test clicked it, but the code that was generated didn't use builders syntax.